### DISCUSSION TITLE
Last open file, text replacement behaviour

### DISCUSSION THREAD

**Discussion by octvs** [Xed-Editor/Xed-Editor#70] [Q&A] [answered]
Hey there, 

Nice app, I have some questions:

- Feature requests:
  - Regarding issue #14, is "remembering last edited file" on the roadmap?
  - How about auto-save?
- Bugs (or weird behaviours):
    - Failing text-suggestion when it is corrective
      - Using default google keyboard on Pixel, when the action is only about
        adding letters everything works, but if it tries to replace/remove some
        it fails. 
      - For example for the word `shelling`:
        - Type `shelli` press on correct suggestion works as intended. 
        - Type `shelking` and press on suggestion it doesn't revert it back to
          `shelling`.
    - File shows always as "changed"
      - The editor thinks there are changes even if I remove all changes
        manually, or pressed undo as much as it allows me to. Each time I get
        the start on the file name, and the pop-up asking me to save even w/o
        any changes.

Let me know if some were intended, not reproduced, or deserves a separate
issue. 

Cheers!

**Comment by RohitKushvaha01** [comment, marked as answer]
> Hey there, 
> 
> Nice app, I have some questions:
> 
> - Feature requests:
>   - Regarding issue #14, is "remembering last edited file" on the roadmap?
>   - How about auto-save?
> - Bugs (or weird behaviours):
>     - Failing text-suggestion when it is corrective
>       - Using default google keyboard on Pixel, when the action is only about
>         adding letters everything works, but if it tries to replace/remove some
>         it fails. 
>       - For example for the word `shelling`:
>         - Type `shelli` press on correct suggestion works as intended. 
>         - Type `shelking` and press on suggestion it doesn't revert it back to
>           `shelling`.
>     - File shows always as "changed"
>       - The editor thinks there are changes even if I remove all changes
>         manually, or pressed undo as much as it allows me to. Each time I get
>         the start on the file name, and the pop-up asking me to save even w/o
>         any changes.
> 
> Let me know if some were intended, not reproduced, or deserves a separate
> issue. 
> 
> Cheers!
> 

Hi there,

Thank you for your feedback and questions!

**Feature Requests:**
- **Remembering last edited file**: Yes, this feature is on the roadmap.
- **Auto-save**: I am currently working on implementing auto-save.

**Bugs/Weird Behaviours:**
- **Failing text-suggestion**:  I'll investigate it further.
- **File shows always as "changed"**: This is a known issue. The editor sometimes incorrectly marks files as changed, While this is recognized, it's currently a low priority to fix.

If you have any more details or encounter other issues, please let me know. 

Cheers!

**Comment by octvs** [comment]
> If you have any more details or encounter other issues, please let me know. 

Not for now, thanks for the rapid response. 

Keep up the good work!
